scalafix-core/shared/src/main/scala/scalafix/internal/rule/ProcedureSyntax.scala

```diff
@@ -8,7 +8,10 @@ import scalafix.Patch
 import scalafix.rule.Rule
 import scalafix.rule.RuleCtx
 
-case object ProcedureSyntax extends Rule("ProcedureSyntax") {
+case object ProcedureSyntax
+    extends Rule(
+      "ProcedureSyntax",
+      "Rule that rewrites special procedure syntax definitions to normal syntax for Dotty support.") {
```

Comment:
Rewrite that inserts explicit `: Unit =` for soon-to-be-deprecated procedure syntax `def foo { ... }`